README.rst

```diff
@@ -43,13 +43,39 @@ libraries in the ``pep0`` directory.
 Checking PEP formatting and rendering
 =====================================
 
-Do not commit changes with bad formatting.  To check the formatting of
-a PEP, use the Makefile.  In particular, to generate HTML for PEP 9999,
-your source code should be in ``pep-9999.rst`` and the HTML will be
-generated to ``pep-9999.html`` by the command ``make pep-9999.html``.
-The default Make target generates HTML for all PEPs.
+Please don't commit changes with reStructuredText syntax errors that cause PEP
+generation to fail, or result in major rendering defects relative to what you
+intend. To check building the HTML output for your PEP (for example, PEP 12)
+using the current default docutils-based system, run the ``pep2html.py`` script
+with your PEP source file as its argument; e.g. for PEP 12,
 
-If you don't have Make, use the ``pep2html.py`` script directly.
+.. code-block:: console
+
+    python pep2html.py pep-0012.rst
+
+If you're on a system with ``make``, you can instead execute, e.g.,
+
+.. code-block:: console
+
+    make pep-0012.rst
+
+To generate HTML for all the PEPs, run the script/``make`` without a PEP
+file argument.
+
+By default, this will output a file (e.g. ``pep-0012.html``) in the root
+directory, which you can view to see the HTML output of your PEP.
+Note that the custom CSS stylesheet is not used by default, so
+the PEP will look rather plain, but all the basic formatting produced by the
+reStructuredText syntax in your source file should be visible.
+
+You can also view your PEP locally with the Sphinx-based builder,
+which will show the PEP exactly as it will appear on the preview
+of the new rendering system proposed in :pep:`676`;
+see `Rendering PEPs with Sphinx`_ for details.
+
+Finally, you can check for and fix common linting and spelling issues,
+either on-demand or automatically as you commit, with our pre-commit suite.
+See the `Contributing Guide <./CONTRIBUTING.rst>`_ for details.
```

Comment:
Would this link also work for the default rendering of the README in the index? I generally prefer relative links, but absolute may be better here if the previous sentence is false.

A

Reply:
Yup, it works; I had the same thought and had specifically tested it to make sure.

Reply:
My policy in the past has generally been to provide the full link, so it is still usable if the file is being accessed from somewhere other than the root of the GitHub page, but as it is likely a small use-case and another link here used the same syntax, I used a relative link here.

Reply:
Fab! Great minds & all that...

A